config/initializers/registers_client.rb

```diff
@@ -1 +0,0 @@
-@@registers_client ||= RegistersClient::RegisterClientManager.new(cache_duration: 600)
```

Comment:
Related to my other comment in the PopulateDatabase, we may need to add this back in so that we don't re-download all register data each time the job is run.